app/views/maestros/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<ul>
+  <% @maestros.each do |maestro| %>
+    <li>
+      <% if maestro.favorite %> * <% end %>
+      <b><%= maestro.artist %></b>
```

Comment:
HTML nowadays uses the `strong` tag instead of `b` to denote when something should be emboldened. It is a semantics thing. `b` and `font-weight: bold` literally increases the pixels and hue of the items, whereas strong does that _and_ tells other types of screenreaders/parsers that "hey! this is important!"